app/src/main/res/values-de/strings.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="format_minutes">%1$dm</string>
+    <string name="format_hours">%1$dh</string>
+    <string name="format_days">%1$dd</string>
+    <string name="format_time_left">%1$s left</string>
```

Comment:
I don't think that these format strings are clear to you, so let me explain: this would result in "mm:ss left", where `mm` are the minutes and `ss` seconds: `time_left`.

This means that this string still needs to be translated.

Reply:
Okay, I did it.

Reply:
I don't quite understand what I'm supposed to translate here.

Reply:
The `%1$d` or `%1$s` are arguments from code. So `format_time_left` gets formatted to the given time + ` left`. This is English, so it needs to be translated to German.

Reply:
The translations are still in English...

Reply:
Let me give you examples of what the `format_time_left` means in the actual interface:

<img src="https://github.com/25huizengek1/ViMusic/assets/50515369/20613ef2-899b-49b9-bdca-8cc33594563e" width="30%" />

And the `format_seconds`, `format_minutes` and `format_hours` are used here (when sorting on play time in the Songs / Local tab):

<img src="https://github.com/25huizengek1/ViMusic/assets/50515369/55558d10-4fb4-4436-b7b3-4dc0bc5f0524" width="30%" />

Reply:
Yes, okay, I think I understand.
But then it's just numbers, do you have to put them over at all?

Reply:
Well yes, that is what the `%1$d` / `%1$s` part is for. For example: 20 hours gets formatted to `20h` in English, because the `h` stands for 'hours'. So yes, you do have to translate this to German.

To translate this, you need to ask yourself the following questions:

- What is "hours" in German? --> put a short abbreviated form after the `%1$d`
- The same, but with minutes
- The same, but with seconds
- What does it mean to have time remaining in German: `format_time_left` is in English "(time) left", but in Dutch "nog (time)", so do the same but in German (`%1$s` stands for the time)

If I'm still not clear and you don't know how to put it in the strings.xml file, you can just answer these questions to me and I'll push the changes accordingly, as I have push access for `siggi1984:patch-22`.

Reply:
Everything should be correct now, sorry and thank you for your patience.